liblangutil/EVMVersion.h

```diff
@@ -60,10 +60,11 @@ class EVMVersion:
 	static EVMVersion paris() { return {Version::Paris}; }
 	static EVMVersion shanghai() { return {Version::Shanghai}; }
 	static EVMVersion cancun() { return {Version::Cancun}; }
+	static EVMVersion prague() { return {Version::Prague}; }
 
 	static std::optional<EVMVersion> fromString(std::string const& _version)
 	{
-		for (auto const& v: {homestead(), tangerineWhistle(), spuriousDragon(), byzantium(), constantinople(), petersburg(), istanbul(), berlin(), london(), paris(), shanghai(), cancun()})
+		for (auto const& v: {homestead(), tangerineWhistle(), spuriousDragon(), byzantium(), constantinople(), petersburg(), istanbul(), berlin(), london(), paris(), shanghai(), cancun(), prague()})
```

Comment:
We should finally break it up into one item per line. Same for the `Version` enum.